packages/rate-limit/rate-limit.js

```diff
@@ -133,7 +143,10 @@ RateLimiter.prototype.check = function (input) {
   var reply = {
     allowed: true,
     timeToReset: 0,
-    numInvocationsLeft: Infinity
+    numInvocationsLeft: Infinity,
+    userId: input.userId,
+    clientAddress: input.clientAddress,
+    connectionId: input.connectionId,
```

Comment:
Just want to confirm I was indeed interested in getting the name and type. One use case is described in https://github.com/meteor/meteor/issues/5541 , namely only banning a user for exceeding certain specific calls, but I can imagine it might be useful for other purposes too.